trucks.js

```diff
@@ -503,9 +505,79 @@ var foodTrucks = [
   }
 ];
 
-// this module should support the following methods:
-// getTrucks() - return all trucks
+// this module should support the following methods: getTrucks() - return all trucks
+var getTrucks = function() {
+  return foodTrucks;
+};
+
 // getTruck(name) - return the truck object matching 'name'
+var getTruck = function(name) {
+  var thisTruck =[];
```

Comment:
This is unnecessary. The method just needs to return one value, and an array is for a list of values. Just return the truck you find that matches and skip putting it into an array.